.github/workflows/e2e.yml

```
@@ -0,0 +1,49 @@
+name: E2E staging
+on:
+  push:
+    branches:
+      - changeset-release/*
+  workflow_dispatch:
+
+concurrency:
+  group: ${{ github.workflow }}-${{ github.ref }}
+
+jobs:
+  e2e:
+    runs-on: ubuntu-latest
```

I recommend freezing the version to avoid unintended breaking changes:
```suggestion
    runs-on: ubuntu-22.04
```